web/themes/joinup/prototype/sass/components/_fieldset.sass

```diff
@@ -16,31 +19,35 @@
     margin-bottom: 0
 
 .fieldset__field-label
-  margin-top: 0
   font-weight: $font-weight-medium
+  margin-top: 0
 
 .fieldset__title
   margin-top: 0
 
+.fieldset
```

Comment:
This seems to be a repetition of https://github.com/ec-europa/joinup-dev/pull/913/files#diff-a500538f142a2994ab3faa0b48a55e26R7

Also, it seems unnecessary:
![screenshot from 2017 09 25 10 48 25](https://user-images.githubusercontent.com/1923476/30800155-5dcbcf74-a1df-11e7-8f58-4e9fe038cddf.png)

Reply:
Duplicate removed.